base/compiler/abstractinterpretation.jl

```diff
@@ -2041,13 +2049,14 @@ function abstract_call_known(interp::AbstractInterpreter, @nospecialize(f),
         end
     elseif la == 3 && istopfunction(f, :!==)
         # mark !== as exactly a negated call to ===
+        call = abstract_call_gf_by_type(interp, f, ArgInfo(fargs, Any[Const(f), Any, Any]), si, Tuple{typeof(!==),Any,Any}, sv, max_methods)
```

Comment:
```suggestion
        call = abstract_call_gf_by_type(interp, f, ArgInfo(fargs, Any[Const(f), Any, Any]), si, Tuple{typeof(f),Any,Any}, sv, max_methods)
```
and better to move this to the line just above `return call` because this isn't needed when `rty` is known to be `Const|Conditional`?